CHANGELOG.md

```diff
@@ -70,4 +85,7 @@
 
 
 
+\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
```

Comment:
This footer should be removed since it is already at the bottom of the file on line `91`. Our changelog generator adds this every time it runs even though it already exists.